src/main/resources/runner/docker/haskell/Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM haskell:8

```

Comment:
I'll have to run some tests locally but how big are the images with this?

Reply:
IIRC, 200-300 MB, these are the only images of Haskell I could find. Unless we want to create custom Haskell images which don't have as much in them. Which I don't think is possible; it just adds GHC and its tools, stack, and cabal and their respective tools.

Reply:
Wow that's alot. I'll do some digging but I think you might be right. I'll have to start looking into reducing the number of images we're creating soon.